Src/AutoFixture.NUnit3.UnitTest/ThrowingStubFixture.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    /// <summary>
+    /// A <see cref="IFixture"/> for the benefit of creating stubs of <see cref="AutoDataAttribute"/> for unit testing
+    /// </summary>
+    public class ThrowingStubFixture : IFixture
```

Comment:
>AFAICT, this `ThrowingStubFixture` doesn't delegate its implementation.

@ploeh Going point.

Based on that, `ThrowingStubFixture` has actually a very precise name. And I shouldn't have proposed to rename it in the first place. @hackle I apologize for this.